app/Http/Controllers/HomeController.php

```diff
@@ -14,6 +15,7 @@ class HomeController extends Controller
      */
     public function index()
     {
-        return view('home');
+        $jobs = Job::get();
```

Comment:
if you need all of your records the way to go is `Job::all()` but in any event I'd recommend always to use at least some kind of sorting, like `Job::orderBy('something')->get()`